src/style.css

```
@@ -1,3 +1,85 @@
-body {
+:root {
+  --main-color: #00acdc;
+}
+
+.body {
+  margin: 0;
+  padding: 0;
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  font-size: 12px;
+  font-style: normal;
+  background-color: lightgray;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  background-color: #fff;
+  min-width: 1024px;
+  padding: 0 50px;
+}
+
+.logo {
+  margin: auto 0;
+  display: block;
+}
+
+li {
+  list-style-type: none;
+}
+
+a {
+  text-decoration: none;
+  color: #000;
+  display: inline-block;
+}
+
+.nav-link {
+  line-height: 60px;
+  margin-left: 20px;
+}
+
+ul {
+  display: flex;
+
+  align-items: center;
+  height: 100%;
   margin: 0;
+  padding: 0;
+}
+
+.link1 {
```

```suggestion
.link:first-child {
```
use pseudo-class and remove link1 class from the html